README.md

```diff
@@ -100,105 +112,134 @@ print("Training set accuracy = " + str(evaluator.evaluate(predictionAndLabels)))
-processed_image_df = transformer.transform(image_df)
+# Assume we have a compiled and trained Keras model
+model.save('path/to/my/model.h5')
+registerKerasImageUDF("my_custom_keras_model_udf", "path/to/my/model.h5")
 ```
 
-
-
-#### For Keras users
-For applying Keras models in a distributed manner using Spark, [`KerasImageFileTransformer`](link_here)
-works on TensorFlow-backed Keras models. It
-* Internally creates a DataFrame containing a column of images by applying the user-specified image
-loading and processing function to the input DataFrame containing a column of image URIs
-* Loads a Keras model from the given model file path
-* Applies the model to the image DataFrame
-
-The difference in the API from `TFImageTransformer` above stems from the fact that usual Keras
-workflows have very specific ways to load and resize images that are not part of the TensorFlow Graph.
-
-
-To use the transformer, we first need to have a Keras model stored as a file. For this example we'll 
-just save the Keras built-in InceptionV3 model instead of training one.
+If there are further preprocessing steps are required to prepare the images, the user has the option to provide a preprocessing function `preprocessor`. The `preprocessor` converts a file path into a image array. This function is usually introduced in Keras workflow, as in the following example.
 
 ```python
 from keras.applications import InceptionV3
+from sparkdl.udf.keras_image_model import registerKerasImageUDF
 
-model = InceptionV3(weights="imagenet")
-model.save('/tmp/model-full.h5')
-```
+def keras_load_img(fpath):
+    from keras.preprocessing.image import load_img, img_to_array
+    import numpy as np
+    img = load_img(fpath, target_size=(299, 299))
+    return img_to_array(img).astype(np.uint8)
 
-Now on the prediction side, we can do:
+registerKerasImageUDF("my_keras_inception_udf", InceptionV3(weights="imagenet"), keras_load_img)
 
-```python
-from keras.applications.inception_v3 import preprocess_input
-from keras.preprocessing.image import img_to_array, load_img
-import numpy as np
-import os
-from sparkdl import KerasImageFileTransformer
-
-def loadAndPreprocessKerasInceptionV3(uri):
-    # this is a typical way to load and prep images in keras
-    image = img_to_array(load_img(uri, target_size=(299, 299)))
-    image = np.expand_dims(image, axis=0)
-    return preprocess_input(image)
-
-transformer = KerasImageFileTransformer(inputCol="uri", outputCol="predictions",
-                                        modelFile="/tmp/model-full.h5",
-                                        imageLoader=loadAndPreprocessKerasInceptionV3,
-                                        outputMode="vector")
-
-files = [os.path.abspath(os.path.join(dirpath, f)) for f in os.listdir("/data/myimages") if f.endswith('.jpg')]
-uri_df = sqlContext.createDataFrame(files, StringType()).toDF("uri")
-
-final_df = transformer.transform(uri_df)
 ```
 
 
+<a id="releases"></a>
+
 ## Releases:
 
-**TBA**
+**TBA**
```

Comment:
Might as well update with our first release info for 0.1.0.

We could also release 0.1.1 (0.2.0?) now that the UDF creation function is in.